README.md

```diff
@@ -137,30 +137,34 @@ let who = game.whose_turn(); // white/black
 // show a visual representation of chess, through stdout
 game.show_board();
 
-// to select a character
-let character = game.select(Pawn(1, G)); // warns if not legal
+// to select a character //if this engine doesn't provide GUI selection replace select with move
```

Comment:
`select` to check for possible moves(for some future action too ;-) ),
if we don't `select` how to check for possible moves!!

think `select` as of picking a character before moving.

`move` actually moves the character,